expression/src/dependencies.rs

```diff
@@ -31,7 +31,7 @@ impl ExpressionDependencies {
             .current_dir(cargo_workspace.path())
             .arg("build")
             .arg("--release")
-            .arg("--frozen")
+            .arg("--offline")
```

Comment:
how does this work inside the container? we only copy the executable from builder container into the runner container. Have you tested it?

Reply:
[f6a6d1e](https://github.com/geo-engine/geoengine/pull/994/commits/f6a6d1e9ebd3804f6a9f0e62efe919c547bac161)